circle.yml

```diff
@@ -0,0 +1,8 @@
+machine:
+  ruby:
+    version: 2.2.1
```

Comment:
Our apps should be upgraded to 2.2.2 sooner rather than later since we are making API calls over https which is affected by the vulnerability fixed in that release.